vuepress/docs/next/docs/getting-started/entando-cli.md

```diff
@@ -114,16 +116,16 @@ To update ent to the latest version and check for dependencies:
 bash <(curl -L "https://get.entando.org/cli") --update
 ent check-env develop
 ```
->Alternatively, to perform a clean install, delete the `~/.entando` directory with `rm -rf ~/.entando`. Then reinstall the CLI with the instructions above. This will also remove the private copies of JHipster, Entando Blueprint, etc.
+>Alternatively, to perform a clean install, delete the `~/.entando` directory via `rm -rf ~/.entando`. Then reinstall the CLI using the instructions above. This will also remove the private copies of JHipster, Entando Blueprint, etc.
 
 ### Customize Quickstart
-The `ent quickstart` command allows parameters that customize your quickstart environment. These options allow you to modify specific properties of your VM, installation versions and databases.
+The `ent quickstart` command accepts parameters to customize your quickstart environment. These options allow you to modify specific properties of your VM, installation versions and databases.
 
 |Operation |Syntax|Description|
 |:--|:--|:--|
 |ent quickstart| ent quickstart --vm-reuse=YOUR-EXISTING-VM| Reuse an existing VM
 ||ent quickstart --release="v7.2.0" | Use a specific release version for the install
```

Comment:
Existing, we probably shouldn't use an example of 7.2.0 which doesn't exist yet. Either we should use the current release, or make it generic. Probably v7.1.0 is best.